addons/account/static/src/components/many2many_tags_banks/many2many_tags_banks.js

```diff
@@ -12,13 +13,15 @@ export class FieldMany2ManyTagsBanksTagsList extends TagsList {
 }
 
 export class FieldMany2ManyTagsBanks extends Many2ManyTagsFieldColorEditable {
+    static template = "FieldMany2ManyTagsBanks";
```

Comment:
Good practice to prefix the template name
```suggestion
    static template = "account.FieldMany2ManyTagsBanks";
```

Reply:
I still think it's a good practice to add on the new one :smile: And in the xml also yeppp :smile: